NAMESPACE

```diff
@@ -146,4 +146,11 @@ export(weights_create)
 export(weights_create_bids)
 import(ggplot2)
 importFrom(dplyr,"%>%")
+importFrom(httr2,req_body_json)
```

Comment:
This file is auto-generated, so as soon as I generate a new one, these changes will be lost. But I can fix this: I'll merge to Development and edit. If you look at the other functions, we typically just use `httr2::req_body_json`. The `usethis` and `devtools` package simplify doing all this finnicky stuff.

Reply:
So it "should" be loading these from the `DESCRIPTION` file? I think I added this b/c it wouldn't see httr when testing it.